stix_shifter_modules/guardium/stix_translation/json/stix_2_1/qsearch_from_stix_map.json

```diff
@@ -0,0 +1,44 @@
+{
+  "x-ibm-finding": {
+      "fields": {
+        "finding_type": ["datacategory"],
+        "start": ["startTime"],
+        "end": ["endTime"],
+        "database_name": ["Database"],
+        "dst_device": ["Database"]
+      }
+    },
+    "ipv4-addr": {
+      "fields": {
+        "value": ["ClientIP", "Server"],
+        "src_ip": ["ClientIP"],
+        "dst_ip": ["Server"]
+      }
+    },
+    "ipv6-addr": {
+      "fields": {
+        "value": ["ClientIP", "Server"],
+        "src_ip": ["ClientIP"],
+        "dst_ip": ["Server"]
+      }
+    },
+    "network-traffic": {
+      "fields": {
+        "dst_ref.value": ["Server"],
+        "src_ref.value":["ClientIP"]
+      }
+    },
+     "user-account": {
+      "fields": {
+        "value": ["DB User", "OS User"],
```

Comment:
There is no `value` property on the user-account object so that should be removed. https://docs.oasis-open.org/cti/stix/v2.1/os/stix-v2.1-os.html#_hpppnm86a1jm